tests/check/check_test.go

```diff
@@ -54,25 +49,30 @@ func testCheck(t *testing.T, engine string) {
 	require.NoError(t, err)
 	defer conn.Close()
 
-	RunSchema1_1CheckTests(t, pb.NewOpenFGAServiceClient(conn))
-	RunSchema1_0CheckTests(t, pb.NewOpenFGAServiceClient(conn))
+	t.Run("RunAll", func(t *testing.T) {
+		t.Run("testCheck", func(t *testing.T) {
+			t.Parallel()
+			testCheck(t, conn)
+		})
+		t.Run("testBadAuthModelID", func(t *testing.T) {
+			t.Parallel()
+			testBadAuthModelID(t, conn)
+		})
+	})
 }
 
-func testBadAuthModelID(t *testing.T, engine string) {
-
-	cfg := run.MustDefaultConfigWithRandomPorts()
-	cfg.Log.Level = "none"
-	cfg.Datastore.Engine = engine
-
-	cancel := tests.StartServer(t, cfg)
-	defer cancel()
+func testCheck(t *testing.T, conn *grpc.ClientConn) {
```

Comment:
Please refactor this a bit so that the composition will allow us to import a single test entrypoint in Sandcastle. For example, like we talked about,

```
package check
...

func RunAllTests(t *testing.T, client CheckTestClientInterface) {
    t.Run("RunAllTests", func(t *testing.T) {
        t.Run("Check", ...)
        t.Run("BadAuthModelID", ...)
    })
}

func testCheck(t *testing.T, client CheckTestClientInterface) {
    t.Run("Schema1_1", func(t *testing.T) {
        t.Parallel()
        schema1_1CheckTests(t, client)
    }
    ...
```

This way we only have to import `RunAllTests` to run all of the Check tests, instead of having to be careful to run each and every one of them.